projects/lib-shared-modules/src/lib/components/dialog-model/dialog-model.component.scss

```diff
@@ -0,0 +1,12 @@
+@import  "../../../../../self-creation-portal/src/styles.scss";
+mat-label{
+    padding-bottom: 18px;
+}
+.btn {
+    width: 90px;
```

Comment:
why button have limited width, and how does it will work for different screen sizes ?